Remotion/Web/Development.WebTesting.IntegrationTests/ScreenshotTest.Shared.cs

```diff
@@ -329,6 +329,8 @@ public void ResolveBorderElementB ()
           {
             var home = Start();
 
+            if (Helper.MainBrowserSession.Headless)
```

Comment:
In WebTestHelper.CreateNewBrowserSession we already adjust the size, just to a different default. I am interested in the details why we need a 20% larger size here. 

And could we use WebTestHelper.CreateNewBrowserSession with a "correct" window size passed via the tests. 

Right now, it feels more like a workaround than the offiical solution?

Reply:
our default creates a scollbar here, as the site is too long. We could change the default for all headless screenshot tests as well. I thought since we only needed to change to another value in this test, we might want to keep the default.

Reply:
We need a 20% larger size because this page does not fit into the standard size. 

The shared screenshot tests in headless should have a different size from the headful tests. We cannot use the window size override in the screenshot tests, as that is already done in the headful tests. 

As for the CreateNewBrowserSession with correct window size: I'm not sure how I would do that without extracting all logic from the IntegrationTest baseclass

Since this only matters for a single test, IMO it makes sense to just add it the test itself.

Reply:
It's using always the same page in this testfixture, why does the size only matter for one test?

Reply:
Because the site was too small including the test output down below, which added the scrollbar. We do not care about seeing the content of that output, so it can stay hidden.